### DISCUSSION TITLE
强制切片是什么意思

### DISCUSSION THREAD

**Discussion by SonderXiaoming** [Mrs4s/go-cqhttp#1587] [Q&A]
最近好多消息被腾讯风控（尤其是贵族决斗之类的）
我开启强制切片貌似就好了
（也没看见分两条发送，还是一条信息）
这个是什么意思
貌似除了不被风控就是我发现用色图插件时可能标题和图片分两段发送（这又导致了自动撤回出bug）

**Comment by tt9891** [comment]
同问，我的被风控了，开了分片也没用
